tools/c7n_org/scripts/orgaccounts.py

```diff
@@ -65,7 +65,7 @@ def main(role, ou, assume, profile, output, regions, active):
             'tags': tags,
             'role': role.format(**a)}
         if regions:
-            ainfo['regions'] = regions
+            ainfo['regions'] = list(regions)
```

Comment:
We’ve actually solved this in c7n.utils, rather than using the yaml.safe_dump we can use c7n.utils yaml_dump which will ignore aliases throughout the entire dump

Reply:
https://github.com/cloud-custodian/cloud-custodian/blob/master/c7n/utils.py#L118

Reply:
safe_dump is unrelated to this issue.

```python
>>> import yaml
>>> x = [1,2,3]
>>> yaml.safe_dump({'a': x, 'b': x})
'a: &id001\n- 1\n- 2\n- 3\nb: *id001\n'
```

Reply:
oh.. ic, we customize the default safe dumper to ignore aliases in c7n/utils